### PR TITLE
fix: failed to create `striped` lv, after add new PV in VG

### PR DESCRIPTION
**Pull request by tangtx3**
fix the issues[#5 ]

### CODE REVIEW

File: pkg/csi/lib/parser.go [hunk 1]
```diff
@@ -407,6 +417,33 @@ func ParseVG(line string) (*VG, error) {
 	}, nil
 }
 
+// ParsePV parse physical volume
```
Comment (peter-wangxu) [inline pkg/csi/lib/parser.go:420]: parse -> `parses`

File: pkg/csi/server/commands.go [hunk 2]
```diff
@@ -186,6 +198,29 @@ func ListVG() ([]*lib.VG, error) {
 	return vgs, nil
 }
 
+// ListPV get pv info in vg
+func ListPV(vgName string) ([]*lib.PV, error) {
+	args := []string{localtype.NsenterCmd, "pvs", "--units=b", "--separator=\"<:SEP:>\"", "--nosuffix", "--noheadings",
```
Comment (peter-wangxu) [inline pkg/csi/server/commands.go:203]: sparator`<:SEP:>` should use constant

**Review by peter-wangxu — CHANGES_REQUESTED** [review]
small nits

File: pkg/csi/lib/parser.go [hunk 3]
```diff
@@ -407,6 +417,33 @@ func ParseVG(line string) (*VG, error) {
...
 
+// ParsePV parses physical volume
+func ParsePV(line string) (*PV, error) {
+	// pvs --units=b --separator="<:SEP:>" --nosuffix --noheadings -o pv_name,pv_size,pv_free,pv_uuid,pv_tags,vg_name -S vg_name=<vg> --nameprefixes -a
+	fields, err := parse(line, 6)
+	if err != nil {
+		return nil, err
+	}
+
+	size, err := strconv.ParseUint(fields["LVM2_PV_SIZE"], 10, 64)
```
Comment (TheBeatles1994) [inline pkg/csi/lib/parser.go:428]: Those can also be expressed as a constant: 
- LVM2_PV_NAME
- LVM2_PV_SIZE
- LVM2_PV_UUID
- LVM2_PV_TAGS

**Pull request merged**